vuestorefront/magento2
# fix(theme, composable): expired user token issue

## Description
- add proper handling of unauthorized requests

## Related Issue
#198 

## Motivation and Context
-

## How Has This Been Tested?

- log-in as a customer
- go to the customer account
- open my account details and change user data
- remove vsf-customer cookie
- press update button
- as a result customer should be redirected to the homepage and appropriate message should be displayed as a warning notification

Similar scenarios can be executed for any customer-token based requests like adding to the cart, removing, changing password, etc.

## Screenshots (if appropriate):

## Types of changes


## Checklist:
